plugin-server/src/main/ingestion-queues/batch-processing/each-batch-ingestion.ts

```diff
@@ -22,6 +24,30 @@ export async function eachMessageIngestion(message: KafkaMessage, queue: Ingesti
         return
     }
 
+    if (
+        !queue.topics().topics.includes(KAFKA_EVENTS_PLUGIN_INGESTION_OVERFLOW) &&
```

Comment:
This won't work in the dev use case or for hobby, as this "queue" will include both topics. We'll need to get it from the `payload.batch.topic` var from the calling function I think.

Reply:
The whole `kafka-queue` abstraction makes this all a little annoying to handle.

An option is that I refactor the consumer away from `kafka-queue` before hand to something more like the `anonymous-buffer-consumer`. Thinking of ways to avoid adding too much to the PR 🤔

Reply:
I did a change to check from `payload.batch.topic` instead :+1: 

I do also think a refactor is in order. However, I don't know if I have the context to do this on my own in a reasonable timeframe. Happy to pair with you on it, or work on changes to this PR after a refactor is done.